src/main/java/stsjorbsmod/patches/SelfExhumeFields.java

```diff
@@ -13,4 +13,5 @@
     public static SpireField<Boolean> selfExhumeOnSnap = new SpireField(() -> false);
     @SuppressWarnings("unchecked")
     public static SpireField<Boolean> selfExhumeOnKill = new SpireField(() -> false);
+    public static SpireField<Boolean> selfExhumeOnTurnX = new SpireField(() -> false);
```

Comment:
This is probably unnecessary; `BottledBurden` can just exhume the specific card it's associated with, rather than tracking the same state in 2 places.